### PR TITLE
Replace message indicating that initialisation has completed

### PR DESCRIPTION
**Pull request by tatsuy**
This is NFC, but I fix it because I see it every time I start SITL.

Copter
image

Sub
image

In Copter, "*" is here.
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_InertialSensor/AP_InertialSensor.cpp#L1233
This is also displayed in Rover. I will leave it because I couldn't decide if it was necessary.

By the way, send_text is used in the other three vehicles. I think it's okay to unify to send_text.

AntennaTracker
https://github.com/ArduPilot/ardupilot/blob/master/AntennaTracker/system.cpp#L82
> gcs().send_text(MAV_SEVERITY_INFO,"Ready to track");

ArduPlane
https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/system.cpp#L214
> gcs().send_text(MAV_SEVERITY_INFO,"Ground start complete");

Rover
https://github.com/ArduPilot/ardupilot/blob/master/Rover/system.cpp#L170
> gcs().send_text(MAV_SEVERITY_INFO, "Ready to drive");

### CODE REVIEW

**Review by peterbarker — APPROVED** [review]
I like it.

I'm a bit worried by the prospect of people pattern-matching this.  But... I also don't think we need to hold up for that.

**Comment by tatsuy** [comment]
I think there is no problem because I haven't changed the string in this PR.

**Comment by peterbarker** [comment]
@tatsuy line-endings... people might be matching on the line endings...

**Comment by rmackay9** [comment]
I think we mostly put the line feed at the end everywhere else so it makes sense to move them to the end.

File: ArduCopter/system.cpp
Comment (tridge) [inline ArduCopter/system.cpp:226]: i think we should make this a send_text with "ArduPilot Ready" in libraries/AP_Vehicle and remove the printf from all vehicles

**Review by tridge — CHANGES_REQUESTED** [review]

**Comment by rmackay9** [comment]
As discussed on the dev call, I think the suggestion from Tridge is a good idea but if it's too difficult I'm happy to merge the Copter change as-is because it's not dangerous and is slightly helpful.

**Comment by tatsuy** [comment]
Thank you for the review!
It's a better idea than mine.
I moved the message to AP_Vehicle.
The timing of displaying it is a bit changed but I think it is more suitable.

I tested this in SITL with all vehicles.

Tracker
```
INITIALISING> Mode INITIALISING
APM: AntennaTracker V1.2.0-dev (b34bdfda)
APM: 3d94fc2306a24139bb795d3de9ba3d0c
Received 785 parameters (ftp)
APM: Barometer 1 calibration complete
APM: ArduPilot Ready
ARMED
AUTO> Mode AUTO
Arming checks disabled
fence breach
APM: EKF2 IMU0 initial yaw alignment complete
APM: EKF2 IMU1 initial yaw alignment complete
```

Copter
```
STABILIZE> Mode STABILIZE
APM: Barometer 1 calibration complete
Init Gyro**APM: ArduCopter V4.1.0-dev (b34bdfda)
APM: 3d94fc2306a24139bb795d3de9ba3d0c
APM: Frame: QUAD
*Received 1154 parameters (ftp)

APM: ArduPilot Ready
APM: SaveWaypoint LOW
APM: EKF2 IMU0 initial yaw alignment complete
APM: EKF2 IMU1 initial yaw alignment complete
```

Plane
```
INITIALISING> Mode INITIALISING
APM: ArduPlane V4.1.0dev (b34bdfda)
APM: 3d94fc2306a24139bb795d3de9ba3d0c
APM: Barometer 1 calibration complete
APM: Airspeed 1 calibration started
APM: ArduPilot Ready
MANUAL> APM: Throttle failsafe off
Received 1199 parameters (ftp)
APM: Airspeed 1 calibrated
Mode MANUAL
APM: GPS 1: detected as u-blox at 230400 baud
APM: EKF2 IMU0 initial yaw alignment complete
APM: EKF2 IMU1 initial yaw alignment complete
```

Sub
```
MANUAL> Mode MANUAL
APM: Barometer 1 calibration complete
APM: ArduPilot Ready
APM: ArduSub V4.1.0dev (b34bdfda)
APM: 3d94fc2306a24139bb795d3de9ba3d0c
Received 1064 parameters (ftp)
Flight battery 100 percent
APM: EKF2 IMU0 initial yaw alignment complete
APM: EKF2 IMU1 initial yaw alignment complete
```

Rover
```
INITIALISING> Mode INITIALISING
APM: ArduRover V4.1.0-dev (b34bdfda)
APM: 3d94fc2306a24139bb795d3de9ba3d0c
Received 1067 parameters (ftp)
APM: Barometer 1 calibration complete
APM: <startup_ground> Ground start
APM: Beginning INS calibration. Do not move vehicle
Init Gyro***
APM: INS: alloc 6144 bytes for ISB (free=131072)
APM: ArduPilot Ready
MANUAL> Mode MANUAL
APM: EKF2 IMU0 initial yaw alignment complete
APM: EKF2 IMU1 initial yaw alignment complete
```

**Comment by rmackay9** [comment]
This looks good to me but I'll leave it to @peterbarker and/or @tridge to make the final decision/merge.

**Review by peterbarker — CHANGES_REQUESTED** [review]
It's definitely going to break https://github.com/ardupilot/ardupilot/blob/master/Tools/scripts/runfliptest.py#L40 :-)

**Comment by peterbarker** [comment]
Should also nudge @jaxxzer on this.

**Comment by tatsuy** [comment]
@peterbarker 
Thank you for your checking.
I also found this.
https://github.com/ArduPilot/ardupilot/blob/master/Tools/LogAnalyzer/DataflashLog.py#L672

I will test these tools with changing the string.

**Comment by tatsuy** [comment]
@peterbarker 
Are these two files still in use?
  Tools/scripts/runfliptest.py
  Tools/scripts/runplanetest.py

If it is used for autotest, I would like to move it to Tools/autotest folder and replace mavproxy.expect with the wait_statustext function.
We already have a test for Flip mode in Tools/autotest/arducopter.py, so I think we don't need it.

About runplanetest.py, it hasn't worked because the message is called by send_text.
https://github.com/ArduPilot/ardupilot/blob/master/Tools/scripts/runplanetest.py#L40

**Comment by peterbarker** [comment]
On Wed, 16 Sep 2020, Tatsuya Yamaguchi wrote:

> @peterbarker
> Are these two files still in use?
> Tools/scripts/runfliptest.py
> Tools/scripts/runplanetest.py

Yes.  Those are tridge's toys, unrelated to autotest :-)

Sorry, I don't think they can be moved - but they will almost certainly 
bitrot by accident.  Glad we caught it this time!

**Comment by tatsuy** [comment]
Should I survive these files?
I can only think of copying the wait_statustext function to these.
Could you give me some advice for merging?

**Comment by peterbarker** [comment]
On Thu, 17 Sep 2020, Tatsuya Yamaguchi wrote:

> I can only think of copying the wait_statustext function to these.

Probably just update the string in these files

**Comment by tatsuy** [comment]
@peterbarker Thank you!
I've realized that I should add APM :.

**Comment by peterbarker** [comment]
> @peterbarker Thank you!
> I've realized that I should add APM :.

No, no you shouldn't :-)

We're trying to get away from `APM` as much as possible as it continues to tie us to outdated, unsupported and now-vexing hardware.

MAVProxy's actually the main offender ATM - we need to change out from `APM` to something else, but need something really short.

Going from `APM: the vehicle wants to be on fire` => `ArduPilot: the vehicle wants to be on fire` is bad (too long, flightstack-specific).

Going from `APM: the fire wants to be on-vehicle` => `AutoPilot: the fire wants to be on-vehicle` is kind of tool long and doesn't scan all that well.

Perhaps we could say, `Vehicle:`?

Anyways.  That's a digression.  `APM` shouldn't happen.

**Comment by tatsuy** [comment]
OK. How about `AP:` ?

**Comment by peterbarker** [comment]
@tatsuy The `APM:` is put there by MAVProxy.  You don't emit it as part of the string, and the expect strings won't need it.

**Comment by tatsuy** [comment]
I misunderstood that an exact match including `APM:` was necessary.
I am sorry for wasting your time.
I've tested that both scripts work fine.

**Review by tridge — APPROVED** [review]

**Pull request merged**
